lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
```

Comment:
When adding structural or styling-minded wrapper elements, we add the ``wrapper-`` prefix. Please change to ``wrapper-hint``.